homeassistant/components/nest/climate_sdm.py

```diff
@@ -112,6 +112,15 @@ def unique_id(self) -> str | None:
         # The API "name" field is a unique device identifier.
         return self._device.name
 
+    @property
+    def available(self) -> bool:
+        """Return entity availability."""
+        if ConnectivityTrait.NAME in self._device.traits:
+            trait: ConnectivityTrait = self._device.traits[ConnectivityTrait.NAME]
+            if trait.status == CONNECTIVITY_TRAIT_OFFLINE:
+                return False
+        return super().available
```

Comment:
I don't think we should call `super()` here since `ThermostatEntity` implements its own logic for availability.
```suggestion
        return True
```

Reply:
Makes sense - updating